packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2435,11 +2435,16 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
   bool get isSemanticBoundary => false;
 
   /// Whether this [RenderObject] makes other [RenderObject]s previously painted
-  /// in the same semantics container unreachable for accessibility purposes.
+  /// within the same semantic boundary unreachable for accessibility purposes.
   ///
   /// If `true` is returned, the [SemanticsNode]s for all siblings and cousins
-  /// of this node, that were painted before this node, are dropped from the
-  /// semantics tree up until a semantic boundary is reached.
+  /// of this node, that are earlier in a depth-first pre-order traversal, are
+  /// dropped from the semantics tree up until a semantic boundary (as defined
+  /// by [isSemanticBoundary]) is reached.
+  ///
+  /// If [isSemanticBoundary] and [isBlockingSemanticsOfPreviouslyPaintedNodes]
+  /// is set on the same node, all previously painted siblings and cousins
+  /// up until the next semantic boundary are dropped.
```

Comment:
This paragraph seems a bit ambiguous. What is the "next semantic boundary"? We're walking a multibranched tree in multiple directions...